torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2860,6 +2864,65 @@ Error PyTorchModelLoader::loadPow(const torch::jit::Node *ptNode) {
   RETURN_ERR(addValueMapping(outputs[0], glowNode->getResult()));
 }
 
+Error PyTorchModelLoader::loadLogicalXor(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 2, outputs, 1));
+
+  glow::NodeValue lhs, rhs;
```

Comment:
There's this function https://github.com/pytorch/glow/blob/9fa5b2d5087896ad178fc11c319ead4bfa006205/torch_glow/src/PyTorchModelLoader.cpp#L2164 that helps load inputs and create node. Maybe we could use that one to replace the replicated parts in the loader (except loadNot since it only has one input).

Reply:
Thanks, I have made the change.